### PR TITLE
Task/and 996 conversation item list

### PR DESCRIPTION
**Pull request by thecombatwombat**

### CODE REVIEW

File: layer-atlas/src/main/java/com/layer/ui/adapters/ConversationItemsAdapter.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                                    ConversationItemFormatter conversationItemFormatter,
+                                    ImageCacheWrapper imageCacheWrapper) {
+        super(context, layerClient, TAG, false);
+        setQuery(query, updateAttributes);
+        mConversationItemFormatter = conversationItemFormatter;
+        mImageCacheWrapper = imageCacheWrapper;
+        mIdentityEventListener = new IdentityRecyclerViewEventListener(this);
+        layerClient.registerEventListener(mIdentityEventListener);
+
+        mIdentityNameFormatter = new IdentityNameFormatterImpl();
```
Comment (andela-wakinsanmi) [inline layer-atlas/src/main/java/com/layer/ui/adapters/ConversationItemsAdapter.java]: Can we pass the IdentityNameFormatterImpl  into the constructor?

File: README.md [hunk 2]
```diff
@@ -96,11 +96,11 @@ The <a href="layer-atlas/src/main/java/com/layer/atlas/AtlasConversationsRecycle
 conversationsList = ((AtlasConversationsRecyclerView) findViewById(R.id.conversations_list))
 	.init(layerClient, picasso)
 	.setOnConversationClickListener(new OnConversationClickListener() {
-		public void onConversationClick(AtlasConversationsAdapter adapter, Conversation conversation) {
+		public void onItemClick(AtlasConversationsAdapter adapter, Conversation conversation) {
```
Comment (thecombatwombat) [inline README.md]: possibly... although this particular call might still have to be made

File: layer-atlas/src/main/java/com/layer/ui/adapters/ConversationItemsAdapter.java (hunk 1 above)
Comment (thecombatwombat) [inline layer-atlas/src/main/java/com/layer/ui/adapters/ConversationItemsAdapter.java]: Good point...

File: layer-atlas/src/main/java/com/layer/ui/messagetypes/CellFactory.java [hunk 3]
```diff
@@ -48,14 +48,14 @@ public CellFactory(int cacheBytes) {
      * @param cellView       ViewGroup to add necessary Message Views to.
      * @param isMe`true`     if this Message was sent by the authenticated user, or `false`.
      * @param layoutInflater Convenience Inflater for inflating layouts.
-     * @return CellHolder with all View references required for binding Messages to Views.
+     * @return CellHolder with all View references required for mBinding Messages to Views.
```
Comment (smpete) [inline layer-atlas/src/main/java/com/layer/ui/messagetypes/CellFactory.java]: This should remain `binding`. Dang refactor problems

File: layer-atlas/src/main/java/com/layer/ui/adapters/ConversationItemsAdapter.java (hunk 1 above)
Comment (thecombatwombat) [inline layer-atlas/src/main/java/com/layer/ui/adapters/ConversationItemsAdapter.java]: Done

File: layer-atlas/src/main/java/com/layer/ui/messagetypes/CellFactory.java (hunk 3 above)
Comment (thecombatwombat) [inline layer-atlas/src/main/java/com/layer/ui/messagetypes/CellFactory.java]: Done

File: README.md (hunk 2 above)
Comment (thecombatwombat) [inline README.md]: Tried this... a binding adapter won't quite do the trick here. Not very cleanly at least. 

Leaving as is for now... can re-visit in the future

**Pull request merged**